File: 2013-06-07/python/exercise1.py
from pyplasm import *
import random

domain = PROD([INTERVALS(20)(30),INTERVALS(21)(30)])

def height(dominio):
	x = dominio[0]
	y = dominio[1]
	z = SIN(x*y)*COS(x*y)

	if x<= 10: dz = x*y*random.random()/500 
	else: dz = x*y*random.random()/100
	
	return [x, y, dz+z]


model = COLOR(BROWN)(MAP(height)(domain))

VIEW(model)

File: 2013-06-07/javascript/exercise1.js
var domain = DOMAIN([[0,20],[0,21]])([50,50])

var height  = function(dominio){
	var x = dominio[0];
	var y = dominio[1];
	var z = SIN(x*y)*COS(x*y);

	//parte pianeggiante + parte montuosa
	if(x<= 10) dz = x*y*Math.random()/500; 
	else dz = x*y*Math.random()/100;
	return [x, y, dz+z];
}

var model = COLOR([139/255,69/255,19/255])(MAP(height)(domain))

DRAW(model)